<launch>
  <param name="use_sim_time" value="true" />

  <arg name="rosbag_path" />

  <arg name="pointcloud_topic" value="lidar_undistortion/pointcloud_corrected" />
  <arg name="rosbag_skip_first_n_sec" value="5" />
  <arg name="debug_mode" value="false" />
  <arg name="show_rviz" value="true" />

  <node pkg="tf" type="static_transform_publisher" name="os1_stuff" args="0 0 0 0 0 1 0 os1_sensor os1_lidar 100" />
  <include file="$(find lidar_undistortion)/launch/os1_undistortion.launch" />

  <include file="$(find voxgraph)/launch/voxgraph_mapper.launch" pass_all_args="true" />
</launch>
